build.gradle

```diff
@@ -52,7 +52,6 @@ allprojects {
         testRuntimeOnly "org.scala-lang.modules:scala-xml_$scalaVersion:1.1.1"
 
         // Logging
-        compile group: 'com.typesafe.scala-logging', name: "scala-logging_$scalaVersion", version: '3.9.2'
```

Comment:
how come you removed it but we're still using it in `package.scala`?

Reply:
this was a typo, the library is already added (on line 45)